packages/flutter/lib/src/gestures/mouse_tracking.dart

```diff
@@ -92,12 +92,23 @@ class MouseTracker extends ChangeNotifier {
   /// Creates a mouse tracker to keep track of mouse locations.
   ///
   /// All of the parameters must not be null.
-  MouseTracker(PointerRouter router, this.annotationFinder)
-      : assert(router != null),
+  MouseTracker(this._router, this.annotationFinder)
+      : assert(_router != null),
         assert(annotationFinder != null) {
-    router.addGlobalRoute(_handleEvent);
+    _router.addGlobalRoute(_handleEvent);
   }
 
+  /// A disposal method that must be called when the MouseTracker is no longer
+  /// needed.
```

Comment:
superclass docs seem adequate, no need to provide an alternative definition here